fivetran/connectors/lambda/fivetran.tf

```diff
@@ -0,0 +1,18 @@
+resource "fivetran_connector" "lambda" {
+  group_id          = var.fivetran_group_id
+  service           = "aws_lambda"
+  sync_frequency    = 60 // min
+  paused            = false
+  pause_after_trial = false
+  run_setup_tests   = true
+
+  destination_schema {
+    name = try(var.connector_name, local.function_name)
+  }
+
+  config {
+    function = aws_lambda_function.main.function_name
+    role_arn = var.lambda_role_arn
+    region   = var.aws_region_code
```

Comment:
https://registry.terraform.io/providers/fivetran/fivetran/latest/docs/resources/connector#region

The documentation mentions `region` and `aws_region_code` ... is there any benefit of using one over the other?

Reply:
I think it is just the different name that fivetran use to define the input variables

eg.
aws_region_code:
https://fivetran.com/docs/rest-api/connectors/config#dynamodb

region:
https://fivetran.com/docs/rest-api/connectors/config#amazonads